modules/sdk-core/src/bitgo/lightning/iLightning.ts

```diff
@@ -0,0 +1,164 @@
+import * as t from 'io-ts';
+
+export interface CreateInvoiceParams {
+  amount: number;
+  memo?: string;
+  expiration?: number;
+}
+
+export interface CreateDepositAddressParams {
+  amount: number;
+  memo?: string;
+  expiration?: number;
+}
+
+export interface GetInvoicesParams {
```

Comment:
Is this going to query our transfers? I'm not sure what this call will be doing.